src/VisualStudio/Core/Def/InheritanceMargin/InheritanceGlyphFactoryProvider.cs

```diff
@@ -36,6 +39,7 @@ internal class InheritanceGlyphFactoryProvider : IGlyphFactoryProvider
         [ImportingConstructor]
         [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
         public InheritanceGlyphFactoryProvider(
+            VisualStudioWorkspaceImpl workspace,
```

Comment:
Rather than importing the internal implementation type, just import VisualStudioWorkspace (which is the public type). It helps when trying to detangle stuff to make it clear in our codebase what is relying on internal implementation and what isn't.